sklearn/metrics/cluster/unsupervised.py

```diff
@@ -207,13 +207,13 @@ def silhouette_samples(X, labels, metric='euclidean', **kwds):
     return sil_samples
 
 
-def calinski_harabaz_score(X, labels):
```

Comment:
please keep something along the lines of

```python
@deprecated(...)
def calinski_harabaz_score(X, labels):
    return calinski_harabasz_score(X, labels)
```

See http://scikit-learn.org/stable/developers/contributing.html#deprecation